<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battleship</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>520</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPassword">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignUp">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>270</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtUsername">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignIn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>220</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QPushButton" name="btnSignOut">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGetUsers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>370</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>History</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: blue;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetHistory">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>40</y>
        <width>370</width>
        <height>470</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnQuickMatch">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Quick Match</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnShowRank">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Rank</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlayWithBot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Play with bot</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="matchPage">
     <widget class="QGraphicsView" name="myView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>380</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="enemyView">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>90</y>
        <width>380</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>Your board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>50</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Enemy board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txtStatus">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>600</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: blue;</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOk">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtMe">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txtEnemy">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>480</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSurrender">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>440</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Surrender</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="chattxt">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>399</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listchat">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="replayPage">
     <widget class="QGraphicsView" name="player1View">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>380</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="player2View">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>90</y>
        <width>380</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="txtPlayer1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>Player1's board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txtPlayer2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>480</y>
        <width>380</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Player2's board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>430</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrev">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNext">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>90</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtReplayStatus">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>480</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: blue;</string>
      </property>
      <property name="text">
       <string>Replay status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtUsername</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>btnSignIn</tabstop>
  <tabstop>btnSignUp</tabstop>
  <tabstop>btnSignOut</tabstop>
  <tabstop>btnGetUsers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
